packages/react/README.md

```diff
@@ -11,7 +11,7 @@ Out of the box, the package provides you the following features:
 
 ## Installation
 
-### Use with React Router 5 and 6 (without Data API)
+### Use with React Router 5 and 6 (no Data Routers)
 
 This describes how to setup Faro-React to use with React Router V5 and V6.
```

Comment:
To set up Faro-React with React Router V5 and V6 without Data routers, add the following code snippet to your project. If you use React Router V6 with Data Routers, refer to the [React Router with Data Routers](#react-router-with-data-routers) section.

Reply:
React V5 and 5 do not support data routers. 

* https://reactrouter.com/en/main/upgrading/v6-data
* https://remix.run/blog/remixing-react-router